openapi-generator/src/main/kotlin/org/jellyfin/openapi/builder/extra/FileSpecBuilder.kt

```diff
@@ -0,0 +1,16 @@
+package org.jellyfin.openapi.builder.extra
+
+import com.squareup.kotlinpoet.FileSpec
+import org.jellyfin.openapi.builder.Builder
+import org.jellyfin.openapi.constants.Strings
+import org.jellyfin.openapi.model.JellyFile
+
+class FileSpecBuilder : Builder<JellyFile, FileSpec> {
+	override fun build(data: JellyFile): FileSpec {
+		return FileSpec.builder(data.namespace, data.typeSpec.name!!)
+			.indent("\t")
```

Comment:
:monocle_face:

Reply:
The Kotlin files in this repository use tabs because tabs are superior and spaces suck. You just didn't see the light yet :)

Reply:
:fire: